frontend/src/loglist/LogsTableWrapper.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import LogsTable from "./LogsTable";
+
+export default class LogsTableWrapper extends React.Component {
```

Comment:
We should pick either plural or not for `Log` in our component names
LogApi, LogListPage, LogsTableWrapper, LogsTable are inconsistent
IMO we should make them all singular and rename to LogTableWrapper and LogTable

Reply:
Should I change this for the StreamsTable as well? I was wondering about that in that PR, but no one mentioned anything so I figured it would be fine to do the same here

Reply:
Created issue #340 to address this in a separate PR